clickhouse/db-system-async-metrics.yml

```diff
@@ -0,0 +1,46 @@
+type: db
+data:
+  query: SELECT * FROM asynchronous_metrics;
+  dbUrl: jdbc:clickhouse://${SPM_MONITOR_CLICKHOUSE_DB_HOST_PORT}/system
+  dbDriverClass: ru.yandex.clickhouse.ClickHouseDriver
+  dbUser: ${SPM_MONITOR_CLICKHOUSE_DB_USER}
+  dbPassword: ${SPM_MONITOR_CLICKHOUSE_DB_PASSWORD}
+  #  dbVerticalModel="true" - because output of `SELECT * FROM asynchronous_metrics;` query gives each attribute in its own row
+  # (1st column being attribute name, 2nd column being attribute value)
+  dbVerticalModel: true
+
+observation:
+  # periodically (asynchronously) updated metrics
+  - name: async-metrics
+    metricNamespace: clickhouse
+
+    metric:
+      - name: repl.queue.max.relative.delay
```

Comment:
I think we should typically use hierarchical names like: ```repl.queue.delay.relative.max```, some other metrics below have the same issue (```max.part.count``` would be ```part.count.max```, ```repl.queue.max.absolute.delay``` would be ```repl.queue.delay.absolute.max``` etc).